go/vt/tabletserver/query_splitter.go

```diff
@@ -281,3 +306,27 @@ func (qs *QuerySplitter) parseFloat(pkMinMax *mproto.QueryResult) ([]sqltypes.Va
 	}
 	return boundaries, nil
 }
+
+// TODO(shengzhe): support split based on min, max from the string column.
+func (qs *QuerySplitter) splitBoundariesStringColumn() ([]sqltypes.Value, error) {
+	firstRow := int64(0x0)
+	lastRow := int64(0xFFFFFFFF)
```

Comment:
Can you please document why you picked this constant?

I assume for the same reason you mentioned below: MySQL uses only the most 4 significant bytes for comparison?